tracim/tracim/i18n/fr/LC_MESSAGES/tracim.po

```diff
@@ -1559,6 +1559,18 @@ msgstr "Voir la dernière version"
 msgid "<b>This file is deprecated</b>"
 msgstr "<b>Ce fichier est obsolète</b>"
 
+#: tracim/templates/file/getone.mak:114
+msgid "Preview"
+msgstr "Prévisualisation"
+
+#: tracim/templates/file/getone.mak:136
+msgid "this page"
+msgstr "cette page"
+
+#: tracim/templates/file/getone.mak:143
+msgid "all pages"
+msgstr "toutes les pages"
```

Comment:
theses translation entry aren't used anymore  
https://github.com/tracim/tracim/pull/281/commits/cb7864373a5f04b4439d46018362cbb734bed6e1 removes it